test/e2e/devsandbox-dashboard/README.md

```diff
@@ -0,0 +1,45 @@
+# Developer Sandbox Dashboard E2E Tests
+
+The UI E2E tests are executed against the Developer Sandbox Dashboard running in OpenShift.
+
+*Prerequisites*:
+
+1. You need a OCP cluster
+    - ROSA cluster from ClusterBot will not work since we are not able to modify the OAuth configuration of ROSA clusters created by the ClusterBot.
+2. Ensure you are using Node.js version 22
+    - to easily manage it, you can run `nvm use 22`
+3. Ensure you have `yarn` installed
+4. Make sure you can log in at <https://sso.devsandbox.dev/auth/realms/sandbox-dev/account> using your SSO_USERNAME and SSO_PASSWORD
+5. Make sure you have toolchain resources deployed on your cluster (you can run `make prepare-and-deploy-e2e`)
```

Comment:
I think it would be safe to share the sso test user in the doc, as others might not have that and there is no ability to create a user there if you're not an admin afaik.